build.cake

```diff
@@ -66,38 +62,6 @@ var packageWhitelist = new[] { "ReactiveUI-Testing", "ReactiveUI-Events", "React
 // Define global marcos.
 Action Abort = () => { throw new Exception("a non-recoverable fatal error occurred."); };
 
-Action<string> RestorePackages = (solution) =>
-{
-    NuGetRestore(solution, new NuGetRestoreSettings() { ConfigFile = "./src/.nuget/NuGet.config" });
-};
-
-Action<string, string> Package = (nuspec, basePath) =>
-{
-    CreateDirectory(artifactDirectory);
-
-    Information("Packaging {0} using {1} as the BasePath.", nuspec, basePath);
-
-    NuGetPack(nuspec, new NuGetPackSettings {
-        Authors                  = new [] {"ReactiveUI contributors"},
-        Owners                   = new [] {"xpaulbettsx", "flagbug", "ghuntley", "haacked", "kent.boogaart", "mteper", "moswald", "niik", "onovotny", "rdavisau", "shiftkey"},
-
-        ProjectUrl               = new Uri("http://www.reactiveui.net"),
-        IconUrl                  = new Uri("https://i.imgur.com/7WDbqSy.png"),
-        LicenseUrl               = new Uri("https://opensource.org/licenses/ms-pl.html"),
-        Copyright                = "Copyright (c) ReactiveUI and contributors",
-        RequireLicenseAcceptance = false,
-
-        Version                  = nugetVersion,
-        Tags                     = new [] {"mvvm", "reactiveui", "Rx", "Reactive Extensions", "Observable", "LINQ", "Events", "xamarin", "android", "ios", "forms", "monodroid", "monotouch", "xamarin.android", "xamarin.ios", "xamarin.forms", "wpf", "winforms", "uwp", "winrt", "net45", "netcore", "wp", "wpdev", "windowsphone", "windowsstore"},
-        ReleaseNotes             = new [] { string.Format("{0}/releases", githubUrl) },
-
-        Symbols                  = false,
-        Verbosity                = NuGetVerbosity.Detailed,
-        OutputDirectory          = artifactDirectory,
-        BasePath                 = basePath,
-    });
-};
-
 Action<string> SourceLink = (solutionFileName) =>
 {
     GitLink("./", new GitLinkSettings() {
```

Comment:
There's a number of GitLink errors [here](https://ci.appveyor.com/project/ghuntley/reactiveui/build/538.Branch.pull/1332/merge.Sha.e0bf30c75068aadea285a7604e40d1d405dee16c#L1951) due to what I suspect is the change in project format:


> Failed to load project 'C:\projects\reactiveui\.\src\ReactiveUI\ReactiveUI.csproj': The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format, please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.  C:\projects\reactiveui\src\ReactiveUI\ReactiveUI.csproj